### PR TITLE
Edits for Sections 2.6 to 3.3.4, inclusive

### PR DESCRIPTION
**Pull request by pllim**
Edits for Sections 2.6 to 3.3.4, inclusive. Follow up of #49. Remaining sections will be submitted as separate pull request(s) when I next get the chance.

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -447,7 +446,7 @@ \subsubsection{Logarithmic units and magnitudes}
 \subsubsection{Defining functions that require quantities}
         When writing code or
         functions that expect \texttt{Quantity} objects, we often want to
-        enforce that the input units have the correct type.
+        enforce that the input units have the correct physical type.
```
Comment (pllim) [inline main.tex:449]: Note: "type" is too cryptic. I *think* physical type is accurate here?

File: main.tex [hunk 2]
```diff
@@ -580,16 +579,16 @@ \subsubsection{Local Earth coordinate frames}
...
     As the default representation for most frames is spherical, most of the
     \texttt{Frame} classes expect proper motion and radial velocity components
     to specify the velocity information.
     The names of the proper motion components all start with \texttt{pm} and
     adopt the same longitude and latitude names as the positional components.
     Transforming coordinates with velocity data is also supported, but in some
     cases the transformed velocity components have limited accuracy because the
-    transformations are done numerically.
-    The visualization of the coordinate frame transform graph highlights which
+    transformations are done numerically instead of analytically.
```
Comment (pllim) [inline main.tex:589]: @eteq or @adrn might need confirm whether changes here are correct. My edits are intended to clarify things for readers.

File: main.tex [hunk 3]
```diff
@@ -625,9 +624,11 @@ \subsubsection{Solar System Ephemerides}
...
+no package can be guaranteed to implement all transformations to arbitrary precision and some transformations are sometimes subject to interpretation of standards (in particular in the case of Galactic coordinates), we do not designate any of the existing packages as the ``ground truth'' but instead compare each tool to all other tools. The benchmarks are thus useful beyond the \astropy project since they allow all of the tools to be compared to all other tools. The tools included in the benchmark at the moment include the \astropypkg core package, Kapteyn \citep{kapteyn}, NOVAS \citep{novas}, PALpy \citep{pal}, PyAST \citep[a wrapper for AST, described in][]{ast}, PyTPM\footnote{\url{https://github.com/phn/pytpm}}, PyEphem \citep{pyephem}, and pySLALIB \citep[a Python wrapper for SLALIB, described in][]{slalib}.
```
Comment (pllim) [inline main.tex]: Note: The diff here looks more drastic than it really is. I only did minor edits, but this is a very long single line, so here we are.

File: main.tex [hunk 4]
```diff
@@ -625,9 +624,11 @@ \subsubsection{Solar System Ephemerides}
...
+
+\tablename~\ref{tab:coordinate_benchmarks} gives an example of the relative accuracy of the conversion from FK4 to Galactic coordinates for all pairs of tools. This shows, for example, that \astropy, Kapteyn, and PyTPM agree perfectly (to the precision shown), while PALpy, pySLALIB, and PyAST also agree perfectly amongst themselves. However, there is an offset of around 0.2\arcsec\space with the former three packages. Finally, PyEphem disagrees with other packages by 0.4--0.8\arcsec. These values are only meant to be illustrative and will change over time as the benchmarks are refined and the packages updated.
```
Comment (pllim) [inline main.tex]: Note: I thought it would be clearer to have the Table stuff as a separate paragraph.

File: main.tex (hunk 3 above)
Comment (bsipocz) [reply main.tex]: feel free to wrap lines, we didn't do it when migrated over saying that whoever touches those lines can do it then.

File: main.tex (hunk 1 above)
Comment (adrn) [reply main.tex:449]: Yep!

File: main.tex [hunk 5]
```diff
@@ -523,11 +522,11 @@ \subsection{Coordinates}
 (\texttt{ra}) and declination (\texttt{dec}), whereas in the Galactic frame, the
 spherical angles are Galactic longitude (\texttt{l}) and latitude (\texttt{b}).
 Each of the \texttt{Frame} classes define their own component names and
```
Comment (adrn) [inline main.tex]: Hm, don't like the plural "classes" - maybe it should be
```raw
Each \texttt{Frame} class defines its own component names and preferred \texttt{Representation} class.
```

File: main.tex [hunk 6]
```diff
@@ -580,16 +579,16 @@ \subsubsection{Local Earth coordinate frames}
 
 \subsubsection{Proper motion and velocity transformations}
     In addition to positional coordinate data, the \texttt{Frame} classes now
-    also support velocity data.
+    also support velocity.
```
Comment (adrn) [inline main.tex]: I think data should be kept.

**Comment by pllim** [comment]
Thanks for the feedback, I think I have addressed them.

**Comment by adrn** [comment]
LGTM now - thanks!

**Pull request merged**